opm/upscaling/CornerpointChopper.hpp

```diff
@@ -40,12 +40,9 @@ namespace Opm
     class CornerPointChopper
     {
     public:
-        CornerPointChopper(const std::string& file)
+        CornerPointChopper(const std::string& file) :
+          deck_(parser_.parseFile(file, parseContext_))
```

Comment:
Do we actually need a `parser_` and a `parseContext_` data member.  Can we not do
```C++
deck_(Parser{}.parseFile(file, ParseContext{}))
```
instead?

Moreover, this is an opportunity to also initialise the `metricUnits_` in the initialiser list.  I think we should do that too.

Reply:
i guess we do not need the members, i was just being safe. i'll check. and i'll move the units.

Reply:
updated.